gcp-resources/build.gradle.kts

```diff
@@ -0,0 +1,30 @@
+plugins {
+  id("otel.java-conventions")
+
+  id("otel.publish-conventions")
+}
+
+description = "OpenTelemetry GCP Resources Support"
+otelJava.moduleName.set("io.opentelemetry.contrib.gcp.resource")
+
+dependencies {
+  api("io.opentelemetry:opentelemetry-api")
+  api("io.opentelemetry:opentelemetry-sdk")
+
+  // Provides GCP resource detection support
+  implementation("com.google.cloud.opentelemetry:detector-resources-support:0.27.0")
```

Comment:
[question] would this be automatically updated through dependabot (or similar) when there is a new release ?

Reply:
renovate will auto-update this by default